cli/tests/test_resolve_command.rs

```diff
@@ -522,32 +522,23 @@ fn test_simplify_conflict_sides() {
     fileA    4-sided conflict
     fileB    4-sided conflict
     "###);
-    // TODO: The conflict should be simplified before being materialized.
```

Comment:
I would add an explicit comment:

"Even thought the tree-level conflict is a 4-sided conflict, each file is materialized as a 2-sided conflict".

Also, I'd demo that the tree-level conflict is 4-sided if possible.

Reply:
> Also, I'd demo that the tree-level conflict is 4-sided if possible.

Any suggestions on how to do that? I was thinking `jj debug tree`, but that actually recursively goes into each tree and shows a path-level conflict instead. Would that still be fine?

Reply:
Hmm... I think there shouldn't be a problem doing it in `test_local_working_copy::test_materialize_conflicted_files`.

Here, it might not be worth it. You might be able to manipulate `jj debug operation` or add options to it or add options to `jj debug tree` if you wanted to.